prov/efa/src/efa_domain.c

```diff
@@ -190,7 +190,7 @@ int efa_domain_open(struct fid_fabric *fabric_fid, struct fi_info *info,
 		ret = err;
 		goto err_free;
 	}
-
+	efa_domain->util_domain.av_type = FI_AV_TABLE;
```

Comment:
I do not see this line before the info code refactor.. Why do we need it now?

Reply:
This happens with the domain squash.

Previously, user call fi_domain, which call rxr_domain_open, rxr_domain call efa_domain_open with a prov_info not user info. hence this line is not needed.

Now that rxr_domain_open has been merged into efa_domain_open, and efa_domain_open directly work on the info from user, therefore this change is needed.